### ISSUE TITLE
Investigate use of cores

### ISSUE THREAD

**Issue by markkerzner** [shmsoft/FreeEed#25]
I processed the sample data, using FreeEed v2.8.4 from the gui, on a 3
year old Dell R900 (4 x Xeon E7310 @ 1.6 Ghz) running CentOS and
Oracle JDK 6 update 27

While it was running, I looked at the output of the top command.

Most of the 2 minutes of run time, just a single core (of the 16
available) was in use (90%+ utilization), although for short periods
up to 4 cores were used (these other 3 with 15% to 40% utilization).
In other words, at least 12 of the 16 cores were idle.

I've never used hadoop before, so any tips as to what I'd change in
the FE source to better utilize the hardware?

thanks .. Jason
